third-party/jemalloc/README

```diff
@@ -0,0 +1,11 @@
+==========================
+jemalloc README for Chapel
+==========================
+
+This copy of jemalloc is being released with Chapel for convenience and
+uses a snapshot from:
```

Comment:
Do we need to mention when the snapshot was taken?  Do they have versions, etc.?  (See next comment also.)